code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -64,8 +64,8 @@
 	icon_state = "c20r"
 	item_state = "c20r"
 	mag_type = /obj/item/ammo_box/magazine/smgm45
-	fire_delay = 2
-	burst_size = 2
+	fire_delay = 4
```

Comment:
fire_delay is only for burst guns, use fire_rate instead

fire_rate = 4

Reply:
pain